flexmeasures/data/models/reporting/pandas_reporter.py

```diff
@@ -87,11 +87,12 @@ def _compute_report(self, **kwargs) -> list[dict[str, Any]]:
 
         resolution: timedelta | None = kwargs.get("resolution", None)
         belief_time: datetime | None = kwargs.get("belief_time", None)
+        belief_horizon: timedelta | None = kwargs.get("belief_horizon", None)
         output: list[dict[str, Any]] = kwargs.get("output")
 
-        # by default, use the minimum resolution among the output sensors
+        # by default, use the minimum resolution among the input sensors
         if resolution is None:
-            resolution = min([o["sensor"].event_resolution for o in output])
+            resolution = min([i["sensor"].event_resolution for i in input])
```

Comment:
Different PR

Reply:
Oh, indeed. I split this bit into three and I confused the branch in this one. I've created the PR from the right branch and I'm closing this one.